src/components/UrlUtilsService.js

```diff
@@ -12,7 +12,7 @@ goog.provide('ga_urlutils_service');
         // from Angular
         // https://github.com/angular/angular.js/blob/master/src/ng/directive/input.js#L3
         var URL_REGEXP =
-            /^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/;
+        /^(ftp|http|https):\/\/(?:\w+(?::\w+)?@)?[^\s/]+(?::\d+)?(?:\/[\w#!:.?+=&%@\-/[\]$'()*,;~]*)?$/;
```

Comment:
This doesn't look like anything in https://github.com/angular/angular.js/blob/master/src/ng/directive/input.js#L3 anymore. So maybe comment should be removed.